src/pages/Account/Login.tsx

```diff
@@ -82,7 +83,9 @@ export const Login: React.FC<LoginProps> = ({
     setFormToDisplay("passwordReset");
   };
 
-  const redirectAfterLogin = () => {};
+  const afterUserIsLoggedIn = () => {
+    trackMixpanelEvent("Login", { worldId: venue.worldId });
```

Comment:
can we track the id or email of the person who is logged in to be able to count unique users?